packages/gatsby/src/services/initialize.ts

```diff
@@ -268,6 +268,15 @@ export async function initialize({
     store.dispatch({
       type: `DELETE_CACHE`,
     })
+
+    // in future this should show which plugin's caches are purged
+    // possibly should also have which plugins had caches
+    telemetry.decorateEvent(`BUILD_END`, {
+      pluginCachesPurged: [`*`],
+    })
+    telemetry.decorateEvent(`DEVELOP_STOP`, {
+      pluginCachesPurged: [`*`],
+    })
```

Comment:
Isn't this just dead code right now? Wouldn't it make more sense to add it when we actually set the value to something meaningful? Or are these going to be set in incremental PR's on a per-plugin basis?

Reply:
I assume @pieh will set it to incremental when he works on selective cache invalidation.